src/EditorFeatures/Test2/Diagnostics/DiagnosticServiceTests.vb

```diff
@@ -1940,8 +1940,15 @@ class MyClass
                 ' Get cached project diagnostics.
                 Dim diagnostics = diagnosticService.GetCachedDiagnosticsAsync(workspace, project.Id).WaitAndGetResult(CancellationToken.None)
 
-                Assert.Equal(1, diagnostics.Count())
-                Assert.Equal(HiddenDiagnosticsCompilationAnalyzer.Descriptor.Id, diagnostics.Single().Id)
+                ' different behavior between v1 and v2. in v2. in v2, solution crawler never creates non-local hidden diagnostics.
```

Comment:
by the way, I am not sure what non-local hidden diagnostic would mean. since those diagnostic will certainly stale, so having special classification on it will be weird since it can have those thing on random part of file if it is non-local hidden with say unnecessary tag